redbot/cogs/trivia/data/lists/hockey.yaml

```diff
@@ -0,0 +1,2097 @@
+- Here Come the Hawks!
+# Blackhawks Trivia by Windy City Hawkey#7814
+Who was the Captain of the 1934 Stanley Cup Champion Blackhawks?:
+- Charlie Gardiner
+- Gardiner
+Who has the NHL record for fastest hat trick scored?:
+- Bill Mosienko
+- Mosienko
+What year did the Blackhawks introduce Chelsea Dagger as their goal song?:
+- 2008
+- 08
+Who score the GWG in 3rd OT of Game 2 of the 2015 Western Conference final?:
+- Marcus Kruger
+- Kruger
+Who has the longest point streak in team History that lasted 26 games?:
+- Patrick Kane
+- Kane
+Who has the most Blackhawk playoff wins as a goalie?:
+- Corey Crawford
+- Crawford
+Who has the most penalty minutes in Blackhawks history?:
+- Chris Chelios
+- Chelios
+When did the ‘Black Hawks’ become the ‘Blackhawks’?:
+- 86 87
+Which Blackhawks defenseman won the Norris Trophy from 1963 through 1966?:
+- Pierre Pilote
+- Pilote
+Who score the famous NO GOAL by head butting the puck in the net?:
+- Andrew Shaw
+- Shaw
+Who won the Norris Trophy in 1982?:
+- Doug Wilson
+- Wilson
+Who sat out in a contract dispute with the Blackhawks and lost his iron man streak, which at the time was the longest stretch in the NHL?:
+- Steve Larmer
+- Larmer
+# Blackhawks Trivia by astrocreep#7211
+Who was voted “worst franchise in sports” by ESPN in 2004?:
+- Chicago Blackhawks
+- Blackhawks
+How many players have scored their 500th goal while playing in Chicago?:
+- 5
+- five
+# Blackhawks Trivia by Gates#9008
+Which legendary Black Hawk Captain famously smoked in the locker room between periods?:
+- Stan Mikita
+- Mikita
+# Blackhawks Trivia by scipio#4358
+What former Blackhawk is now a pilot for American Airlines?:
+- Al Secord
+- Secord
+# Hockey Trivia by aschwan41#0699
+What team tried to kick the Stanley Cup across a canal?:
+- The Ottawa Senators
+Who was drafted 133rd overall in the 1994 NHL Entry Draft?:
+- Daniel Alfredsson
+- Alfredsson
+What team hosted the 1999 NHL Entry Draft?:
+- Boston Bruins
+- Boston
+- Bruins
+What team hosted the 2000 NHL Entry Draft?:
+- Calgary Flames
+- Calgary
+- Flames
+What team hosted the 2001 NHL Entry Draft?:
+- Florida Panthers
+- Florida
+- Panthers
+What team hosted the 2002 NHL Entry Draft?:
+- Toronto Maple Leafs
+- Toronto
+- Maple Leafs
+- Leafs
+What team hosted the 2003 NHL Entry Draft?:
+- Nashville Predators
+- Nashville
+- Predators
+What team hosted the 2004 NHL Entry Draft?:
+- Carolina Hurricanes
+- Carolina
+- Hurricanes
+What team hosted the 2005 NHL Entry Draft?:
+- Ottawa Senators
+- Ottawa
+- Senators
+What team hosted the 2006 NHL Entry Draft?:
+- Vancouver Canucks
+- Vancouver
+- Canucks
+What team hosted the 2007 NHL Entry Draft?:
+- Columbus Blue Jackets
+- Columbus
+- Blue Jackets
+What team hosted the 2008 NHL Entry Draft?:
+- Ottawa Senators
+- Ottawa
+- Senators
+What team hosted the 2009 NHL Entry Draft?:
+- Montreal Canadiens
+- Montreal
+- Canadiens
+What team hosted the 2010 NHL Entry Draft?:
+- Los Angeles Kings
+- LA Kings
+- LA
```

Comment:
Some question before about LA does not have "LA" as a correct response. I'd keep it same-ish.